src/rdkafka_admin.c

```diff
@@ -2737,3 +2964,474 @@ rd_kafka_DescribeConfigs_result_resources (
+                        "when only %d were requested", group_cnt,
+                        rd_list_cnt(&rko_req->rko_u.admin_request.args));
+
+        rko_result = rd_kafka_admin_result_new(rko_req);
+        rd_list_init(&rko_result->rko_u.admin_result.results,
+                     group_cnt,
+                     rd_kafka_group_result_free);
+
+        for (i = 0 ; i < (int)group_cnt ; i++) {
+                rd_kafkap_str_t kgroup;
+                int16_t error_code;
+                rd_kafka_group_result_t *groupres;
+                rd_kafka_DeleteGroup_t skel;
+                int orig_pos;
+                rd_kafka_error_t *error;
+
+                rd_kafka_buf_read_str(reply, &kgroup);
+                rd_kafka_buf_read_i16(reply, &error_code);
+
+                if (error_code) {
+                        error = rd_kafka_error_new(error_code, "%s", rd_kafka_err2str(error_code));
+                } else {
+                        error = rd_kafka_error_new(error_code, NULL);
+                }
+
+                groupres = rd_kafka_group_result_new(kgroup.str,
+                                                     RD_KAFKAP_STR_LEN(&kgroup),
+                                                     error);
+
+                /* As a convenience to the application we insert group result
+                 * in the same order as they were requested. The broker
+                 * does not maintain ordering unfortunately. */
+                skel.group = groupres->group;
+                orig_pos = rd_list_index(&rko_req->rko_u.admin_request.args,
+                                         &skel, rd_kafka_DeleteGroup_cmp);
+                if (orig_pos == -1) {
+                        rd_kafka_group_result_destroy(groupres);
+                        rd_kafka_buf_parse_fail(
+                                reply,
+                                "Broker returned group %.*s that was not "
+                                "included in the original request",
+                                RD_KAFKAP_STR_PR(&kgroup));
+                }
+
+                if (rd_list_elem(&rko_result->rko_u.admin_result.results,
+                                 orig_pos) != NULL) {
+                        rd_kafka_group_result_destroy(groupres);
+                        rd_kafka_buf_parse_fail(
+                                reply,
+                                "Broker returned group %.*s multiple times",
+                                RD_KAFKAP_STR_PR(&kgroup));
+                }
+
+                rd_list_set(&rko_result->rko_u.admin_result.results, orig_pos, groupres);
+        }
+
+        *rko_resultp = rko_result;
+        return RD_KAFKA_RESP_ERR_NO_ERROR;
+
+err_parse:
+        if (rko_result)
+                rd_kafka_op_destroy(rko_result);
+
+        rd_snprintf(errstr, errstr_size,
+                    "DeleteGroups response protocol parse failure: %s",
+                    rd_kafka_err2str(err));
```

Comment:
reply->rkbuf_err

Reply:
This is the same pattern as the other `*Response_parse` functions, should I update them all?